api/docs/v2/storefront/index.yaml

```
@@ -49,6 +49,46 @@ paths:
           $ref: '#/components/responses/403Forbidden'
       security:
         - bearerAuth: []
+    post:
+      description: >-
+        Creates new account
+      tags:
+        - Account
+      operationId: 'Account Creation'
+      requestBody:
+        required: true
+        content:
+          application/vnd.api+json:
+            schema:
+              type: object
+              properties:
+                user:
+                  type: object
+                  properties:
+                    email:
+                      type: string
+                      example: 'john@snow.org'
+                    password:
+                      type: string
+                      example: 'spree123'
+                    password_confirmation:
+                      type: string
+                      example: 'spree123'
+      responses:
+        '200':
+          description: ok
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/Account'
+        '422':
+          description: Validation errors
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      security:
+        - bearerAuth: []
```

```suggestion
        - bearerAuth: []
```

we cannot have bearerauth here because there's no user yet